### PR TITLE
Improved confirmation function

### PR DESCRIPTION
**Pull request by insolvency**
Made yes and no work and use repetition to re-offer the prompt if the user enters an invalid option

### CODE REVIEW

File: main.ps1 [hunk 1]
```diff
@@ -22,7 +22,11 @@ function Confirmation{
  	}
 
 	$option = Read-Host $text "[y/n]"
```
Comment (YummyBacon5) [inline main.ps1]:
```suggestion
	$option = (Read-Host $text "[y/n]").ToLower()
```

**Review by YummyBacon5 — CHANGES_REQUESTED** [review]

File: main.ps1 (hunk 1 above)
Comment (insolvency) [reply main.ps1]: Lowercase is ugly please

Comment (YummyBacon5) [reply main.ps1]: What do you mean

It's for if someone enters YES then the input will be valid

Comment (Zoobdude) [reply main.ps1]: This looks good @YummyBacon5

File: main.ps1 [hunk 2]
```diff
@@ -21,8 +21,12 @@ function Confirmation{
 		return true
```
Comment (YummyBacon5) [inline main.ps1]: Shouldn't this be `$true`?

Also the same with [this variable with `confirmOverride`](https://github.com/likes-gay/win-config/pull/3/files#diff-da68a81f5f0b063b627f4a96b14f7ab30dc3b74a4b3f444306a27f925c88d96fL15).

**Review by YummyBacon5 — CHANGES_REQUESTED** [review]

**Review by Zoobdude — APPROVED** [review]

**Pull request merged**